test.ts

```diff
@@ -46,11 +48,59 @@ https://example.com/a.ts (32B)
 });
 
 Deno.test({
-  name: "createGraph with load - remote modules",
+  name: "createGraph() - root module missing",
+  async fn() {
+    const graph = await createGraph("file:///a/test.ts", {
+      load() {
+        return Promise.resolve(undefined);
+      },
+    });
+    assertEquals(graph.modules.length, 0);
+  },
+});
+
+Deno.test({
+  name: "createGraph() - invalid URL",
+  fn() {
+    return assertThrowsAsync(
+      async () => {
+        await createGraph("./bad.ts");
+      },
+      Error,
+      "relative URL without a base",
+    );
+  },
+});
+
+Deno.test({
+  name: "createGraph() - load rejects",
+  async fn() {
+    const graph = await createGraph("file:///a/test.ts", {
+      load() {
+        return Promise.reject(new Error("something bad happened"));
+      },
+    });
+    assertEquals(graph.modules.length, 0);
+  },
+});
+
+Deno.test({
+  name: "createGraph() - load throws",
+  async fn() {
+    const graph = await createGraph("file:///a/test.ts", {
+      load() {
+        throw new Error("something bad happened");
+      },
+    });
```

Comment:
It's a bit surprising that error thrown in `load` function is not re-thrown from `createGraph`, but I guess that's by design and not mistake?

Reply:
Any errors thrown by load get stored in the module graph slot.  So if you were to do a `graph.get("file:///a/test.ts")` the error would be "re-thrown" at that point.

Reply:
Ah, thanks, I forgot about that